intl/msg_hash_nl.c

```diff
@@ -266,15 +266,15 @@ const char *msg_hash_to_str_nl(enum msg_hash_enums msg)
       case MENU_ENUM_LABEL_VALUE_FRONTEND_COUNTERS:
          return "Frontend Prestatie Tellers";
       case MENU_ENUM_LABEL_VALUE_HELP:
-         return "Help";
+         return "RetroArch Tour";
```

Comment:
Not sure about having "RetroArch" in the description itself. If you're using Lakka, this entry won't apply. Mind just having "Tour"?